js-console/src/main/scala/org/typelevel/log4cats/console/ConsoleLogger.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2018 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.log4cats
+package console
+
+import cats._
+import cats.syntax.all._
+import org.typelevel.log4cats.extras.LogLevel
+import org.typelevel.log4cats.extras.LogLevel._
+
+class ConsoleLogger[F[_]: Applicative](logLevel: Option[LogLevel] = Option(Trace))(implicit
+    ConsoleF: ConsoleF[F]
+) extends SelfAwareStructuredLogger[F] {
```

Comment:
Because `ConsoleF` is now package-private, we cannot leak it here. So either the constructor should become private, or be written in terms of `Sync[F]`

Reply:
Good catch—I'm surprised the compiler doesn't warn about this. Addressed in fd8d912.